src/Stratis.Bitcoin.Features.Consensus.Tests/InitialBlockDownloadTest.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Microsoft.Extensions.Logging;
+using NBitcoin;
+using Stratis.Bitcoin.Base;
+using Stratis.Bitcoin.Configuration;
+using Stratis.Bitcoin.Utilities;
+using Xunit;
+
+namespace Stratis.Bitcoin.Features.Consensus.Tests
+{
+    public class InitialBlockDownloadTest
+    {
+        private readonly ConsensusSettings consensusSettings;
+        private readonly Checkpoints checkpoints;
+        private readonly NodeSettings nodeSettings;
+        private readonly ChainState chainState;
+
+        public InitialBlockDownloadTest()
+        {
+            this.consensusSettings = new ConsensusSettings(NodeSettings.Default(), new LoggerFactory());
+            this.checkpoints = new Checkpoints(Network.Main, this.consensusSettings);
```

Comment:
Maybe introduce a variable to hold the Network instance and pass that around?

Reply:
Done